client/db/types.go

```diff
@@ -378,75 +378,56 @@ func decodeBalance_v0(pushes [][]byte) (*asset.Balance, error) {
 	}, nil
 }
 
-// BalanceSet represents a wallet's balance in various contexts, which includes
-// the zero-conf case, and balances for all asset-supporting exchanges. The
-// separate balances are required because how much is available depends on an
-// exchange's FundConf setting.
-type BalanceSet struct {
-	ZeroConf *asset.Balance            `json:"zeroConf"`
-	XC       map[string]*asset.Balance `json:"xc"`
-	Stamp    time.Time                 `json:"stamp"`
+// Balance represents a wallet's balance in various contexts.
+type Balance struct {
+	asset.Balance
+	Stamp time.Time `json:"stamp"`
 }
 
-// Encode encodes the BalanceSet to a versioned blob.
-func (b *BalanceSet) Encode() []byte {
-	enc := dbBytes{0}.
-		AddData(encodeBalance(b.ZeroConf)).
+// Encode encodes the Balance to a versioned blob.
+func (b *Balance) Encode() []byte {
+	return dbBytes{0}.
+		AddData(encodeAssetBalance(&b.Balance)).
```

Comment:
Fortunately we didn't do the v0->v1 upgrade yet (https://github.com/decred/dcrdex/pull/448), and the v0 test data doesn't change.